### PR TITLE
README

### PR DESCRIPTION
**Pull request by nmacuk**
Added README and license file

### CODE REVIEW

**Comment by PiotrMisiurek** [comment]
@nmacuk The branch is called `license` but contains `readme`.

There is also conflict that must be resolved

Please fix those issues

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Unnamed-Rails-Project
+
+- This project is about creating a token that people can use to have impact on websites such as blogs.
```
Comment (PiotrMisiurek) [inline README.md]:
```
Centralized token-like credits issuance platform for marketing purpose. Give tokens to your audience. Pay them for sharing, comments and other kind of work for you. Use token to build stronger community. Let yout people decide about your work, your content
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+[![Build Status](http://img.shields.io/travis/badges/badgerbadgerbadger.svg?style=flat-square)](https://travis-ci.org/badges/badgerbadgerbadger) [![License](http://img.shields.io/:license-mit-blue.svg?style=flat-square)](http://badges.mit-license.org)
```
Comment (PiotrMisiurek) [inline README.md]: We don't use travis

Comment (nmacuk) [reply README.md]: Now, we do 💃

**Comment by nmacuk** [comment]
I can't resolve conflicts because as far as I see there is a README file that was created automatically through "rails new". I think that Damian has to delete the README file and then I will be able to push my file.

**Comment by nmacuk** [comment]
I added 2 commits with updates to README, please review it and let me know what do you think about it.

File: README.md (hunk 2 above)
Comment (PiotrMisiurek) [reply README.md]: It is not merged yet. Build status in readme should be the part of travis PR

**Comment by PiotrMisiurek** [comment]
The branch is still called license

The branch is not rebased with master. You merged master but this is not something we want to do.

I would create a new branch `readme` from current master and apply relevant commit from this branch wit `git cherry-pick`. That will solve both issues.

In 95% we expect that PR provieds one more commit to master. There can be a lot of commits during work in progress, but finally  at code review level there should be one commit above master. You can achieve it during rebase process (use squash commits option) during rebase process. Call git rebase -i to be able to do this

**Pull request closed without merging**